test/minikube/minikube_base_database_test.go

```diff
@@ -729,7 +795,101 @@ func TestKubernetesRestoreMultipleSMs(t *testing.T) {
 		require.NoError(t, err, "error running SQL: show schema User")
 		require.True(t, strings.Contains(tables, "No tables found in schema "), "Show schema returned: ", tables)
 		testlib.CheckArchives(t, namespaceName, admin0, opt.DbName, 2, 0)
+		checkRestoreRequests(t, namespaceName, admin0, opt.DbName)
+	})
+}
+
+func TestKubernetesRestoreWithStorageGroups(t *testing.T) {
```

Comment:
how long does the full "upgrade suite" take at this point? Currently CircleCi looks like this:
- short 40min
- long 40min
- diagnostics 20 min
- upgrade 40 min

Once we enable these, will the short/basic suite bloat to 1h+?

In parallel, could I ask you to move the B&R tests from the "base database tests" to their own file, please?

Reply:
`minikube.TestKubernetesRestoreDatabase/restoreDatabaseBackwardsCompatibility` adds 74.93s
`minikube.TestKubernetesRestoreWithStorageGroups ` adds 171.83s (measured in bamboo)

As they are now more test coverage on the B&R, I will move the basic tests in `minikube_base_restore_test.go` marked with +short and the other in `minikube_long_restore_test.go`.